### ISSUE TITLE
Replacing numbers only causes "undefined"

### ISSUE THREAD

**Issue by wlaurance** [harthur/replace#31]
I am using replace@0.2.7 to replace part of a phone number.

I have tried a few variations

``` bash
replace 383 343 -rp .
replace "383" "343" -rp .
```

It results in 

```
<p>By Phone: <a href="tel:+1888undefined8400">888.undefined.8400</a>
```

If I add a non digit character to the replacement text, it seems to pick it up correctly.

```
replace 383 343a -rp .
```

```
<p>By Phone: <a href="tel:+1888343a8400">888.343a.8400</a>
```

Perhaps the numbers aren't parsing correctly? I will take a look when I have some time but wanted to report the issue promptly.

**Comment by harthur** [comment]
Thanks for reporting, that's not good.

Looks like it's just in the preview, and it's actually correctly replaced in the file.

**Comment by harthur** [comment]
Fixed w/ 90a1821. Issue was with use of terminal coloring module. Thanks again.
